pkg/controller/master/upgradelog/common.go

```diff
@@ -32,6 +32,35 @@ func upgradeLogReference(upgradeLog *harvesterv1.UpgradeLog) metav1.OwnerReferen
 	}
 }
 
+func getLoggingImages(candidateImages map[string]image, imageList map[string]interface{}) error {
```

Comment:
@starbops Can you check if this function can be used here? https://github.com/harvester/harvester/blob/8feea706be8bc84b8c30b2f6b5e45c3861345a65/pkg/controller/master/setting/support_bundle_image.go#L23